hatasmota/const.py

```diff
@@ -177,13 +177,17 @@
 # #### UNITS OF MEASUREMENT ####
 # Power units
 POWER_WATT: Final = "W"
+POWER_FACTOR: Final = "Cos φ"
```

Comment:
Why with capital letter here? Cosine function name is "cos" in math. Power factor does not have a real unit anyway - it is a [dimensionless number](https://en.wikipedia.org/wiki/Power_factor) but in case giving it a name, IMHO it is more correct to not capitalize it but use just "cos"

Reply:
This PR just move the existing sensor unit from [sensor](https://github.com/thecode/hatasmota/blob/026c0e46e18febbc3293e85f8ea464b0e5a6981a/hatasmota/sensor.py#L173) to a constant. For Home Assistant the name is irrelevant since we use the constant to map it to Home Assistant units (in this case unitless), however there might be other users for this library and renaming the unit will be a breaking change for them.